### PR TITLE
Initial stylelint config

### PR DESCRIPTION
**Pull request by spzm**
Stylelint config that supports css modules syntax, default stylelint config, css properties sorting

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    ],
+    'color-hex-case': 'upper',
+    'declaration-block-no-duplicate-properties': [
+      true,
+      {
+        ignore: 'consecutive-duplicates',
+      }
+    ],
+    'function-comma-space-after': 'always-single-line',
+    'number-leading-zero': 'never',
```
Comment (EduardTrutsyk) [inline index.js]: as I know, for now we are using: `number-leading-zero: always`, or I'm wrong?

Comment (spzm) [reply index.js]: I think you are correct, going to fix ;)

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "travix/rules/node",
+    "travix/rules/style",
+    "travix/rules/variables",
+    "travix/rules/es6"
+  ],
+  "env": {
+      "browser": false,
+      "node": true
+  }
+}
```
Comment (fahad19) [inline .eslintrc]: you can get the common `.editorconfig` file for getting rid of things like this: https://github.com/Travix-International/frint/blob/master/.editorconfig

Comment (spzm) [reply .eslintrc]: done, thanks :+1:

File: index.js (hunk 1 above)
Comment (spzm) [reply index.js]: updated to `always` cause it visually much easy to recognize by developers

**Pull request merged**